modules/llm-cache/tests/kv_state_cache_test.cc

```diff
@@ -41,11 +41,14 @@ std::vector<int> round_4_tokens = {1, 2, 3, 4, 5, 6};
 std::vector<std::vector<int>> tokens_list;
 
 KVStateCacheManager* kv_state_cache_manager;
+Client client;
 
 void init(int dimension, int capacity, int layer, int block_size,
           std::string socket) {
-  kv_state_cache_manager = new KVStateCacheManager(dimension, capacity, layer,
-                                                   block_size, 3, socket);
+  client.Connect(socket);
+  VINEYARD_CHECK_OK(KVStateCacheManager::Make(client, kv_state_cache_manager,
+                                              dimension, capacity, layer,
+                                              block_size, 3));
```

Comment:
Why not `VINEYARD_CHECK_OK` on `client.Connect()`? I promise you would get another warning ...

Reply:
Sorry, this problem was originally addressed in another pr. I fix it now.